common/src/main/java/com/codeforcommunity/email/Emailer.java

```diff
@@ -0,0 +1,75 @@
+package com.codeforcommunity.email;
```

Comment:
Should this be a singleton pattern? I would assume we'd only ever want to send emails from a single host. It would then make sense that the email configs can be read from a secrets file rather than being passed directly in